### DISCUSSION TITLE
Hot reloading not working

### DISCUSSION THREAD

**Discussion by MichaelNZ85** [livewire/livewire#8237] [1. Help]
I'm using Livewire 3.4.9 with Laravel 11 (running on MacOS with Sail), and hot reloading isn't working at all. I created `config/livewire.php` and added `'hotwire' => true` to it, but nothing. 

I tried installing the [vite livewire plugin](https://github.com/def-studio/vite-livewire-plugin) but that didn't work either. I tried Livewire devtools, but still no luck.

Please help me. Livewire looks so good, but the lack of hot reloading really irritates me.

**Comment by PhiloNL** [comment]
There is no native hot reload feature but the Vite plugin you mentioned works fine (using this myself). Make sure to double check the paths it should watch.

**Comment by joshhanley** [comment]
@MichaelNZ85 Yeah @PhiloNL is correct we don't have hot reloading built in, we just use Vite.

Out of curiosity where did you find adding `'hotwire' => true` to the Livewire config? As that's not a thing 😅

**Reply by MichaelNZ85** [reply]
I found it on [this Laracasts thread](https://laracasts.com/discuss/channels/livewire/livewire-v3-beta-hot-reloading-autoreload-hmr)

**Reply by PhiloNL** [reply]
Ah, that reply is generated by AI and it seems to have mixed things up with hotwire.
